cds-feature-attachments/src/main/resources/cds/com.sap.cds/cds-feature-attachments/attachments-annotations.cds

```diff
@@ -34,6 +34,7 @@ annotate Attachments with @UI: {
         {Value: note}
     ]
 } {
+    content    @Core.ContentDisposition: { Filename: fileName, Type: 'inline' };
```

Comment:
ist da nicht das gleiche wie in: https://github.com/cap-java/cds-feature-attachments/blob/a581f3299f5caf52b234c20d4399f75563af2320/cds-feature-attachments/src/main/resources/cds/com.sap.cds/cds-feature-attachments/attachments-annotations.cds#L10

Reply:
Ok, das hatte ich ganz uebersehen. Es hatte bisher aber nicht funktioniert, der fileName wurde nicht uebertragen. Mit dieser Annotation hat es dann geklappt. Die SDM Kollegen haben danach gefragt.

Reply:
Wie ist das mit Annotations bei Aspekten, werden die auch vererbt ? Bisher war diese Annotation beim Aspect MediaData. Hatte das frueher schon mal funktioniert ?

Reply:
ich dachte sie werden vererbt, aber wenn das nicht richtig funktioniert, dann sollten wir alle an den Attachment-Aspect packen, was denkst du?

Reply:
Scheint ein Bug oder Feature im cds compiler zu sein. Nur wenn ich die Anntation nochmal am Attachment Aspekt mache, bekomme ich die Annotations auch im EDMX zu sehen:

```xml
      <Annotations Target="AdminService.Books_covers/content">
        <Annotation Term="Core.ContentDisposition">
          <Record Type="Core.ContentDispositionType">
            <PropertyValue Property="Filename" Path="fileName"/>
            <PropertyValue Property="Type" String="inline"/>
          </Record>
        </Annotation>
        <Annotation Term="Core.MediaType" Path="mimeType"/>
        <Annotation Term="Common.Label" String="{i18n>attachment_content}"/>
      </Annotations>
```
Ich merge deswegen diesen PR und lasse die Annotation auch am MediaData Aspekt stehen.